app/presenters/publishing_api/publication_presenter.rb

```diff
@@ -75,7 +75,20 @@ def body
 
     def documents
       return [] unless item.attachments.any?
-      Whitehall::GovspeakRenderer.new.block_attachments(item.attachments, alternative_format_email)
+      Whitehall::GovspeakRenderer.new.block_attachments(
+        attachments_for_current_locale,
+        alternative_format_email
+      )
+    end
+
+    def attachments_for_current_locale
+      attachments = item.attachments
+      locales_that_match = Array(I18n.locale.to_s)
+      locales_that_match << "" if I18n.locale == I18n.default_locale
```

Comment:
Yep. The default locale is `:en` but helpfully attachments for the default locale in the whitehall DB have `locale == nil`. It looks like they are all `nil` but selecting on `nil` and `en` seemed like a good idea to be safe.